package.json

```diff
@@ -21,10 +21,15 @@
   "scripts": {
     "coverage": "istanbul cover _mocha -x src/model.js --  --compilers js:babel-core/register --recursive -r jsdom-global/register",
     "start": "node example/server.js",
-    "test": "eslint . && mocha --compilers js:babel-core/register --recursive -r jsdom-global/register",
+    "test": "mocha --compilers js:babel-core/register --recursive -r jsdom-global/register",
+    "lint": "eslint .",
```

Comment:
lets add the --ignore-pattern inside the npm script so we save us yet another config file 👍